macos/Sources/Features/QuickTerminal/QuickTerminalSize.swift

```diff
@@ -0,0 +1,81 @@
+import Cocoa
+import GhosttyKit
+
+class QuickTerminalSize {
+    enum Size {
+        case percent(value: Double)
+        case pixel(value: UInt)
+        
+        init?(c_dimension: ghostty_config_quick_terminal_dimension_s) {
+            switch(c_dimension.unit) {
+            case GHOSTTY_QUICK_TERMINAL_PIXEL_UNIT:
+                self = .pixel(value: UInt(c_dimension.value))
```

Comment:
[TODO] Use screen scale (IIRC retina has a scale of 2-3 while standard disply is 1)